### PR TITLE
Update plugins.md

### PR DESCRIPTION
**Pull request by hendrikschneider**
Updated plugin description

### CODE REVIEW

File: docs/_docs/plugins.md [hunk 1]
```diff
@@ -920,7 +920,7 @@ You can find a few useful plugins at the following locations:
 
 #### Other
 
-- [Analytics for Jekyll](https://github.com/hendrikschneider/jekyll-analytics) by Hendrik Schneider: An effortless way to add  various trackers like Google Analytics, Piwik, etc. to your site
+- [Analytics for Jekyll](https://github.com/hendrikschneider/jekyll-analytics) by Hendrik Schneider: An effortless way to add  various trackers like Google Analytics, Piwik, Matomo, MPulse, etc. to your site
```
Comment (ashmaroli) [inline docs/_docs/plugins.md]: - Piwik is now Matomo.. so it should not be Piwik, Matomo
- All docs refer to "MPulse" as "mPulse"

```diff
- trackers like Google Analytics, Piwik, Matomo, MPulse, etc. to your site
+ trackers like Google Analytics, Matomo (Piwik), mPulse, etc. to your site
```

Comment (hendrikschneider) [reply docs/_docs/plugins.md]: I think it is also important to mention Piwik to show backwards compatibility. Would you accept this description?

```
An effortless way to add various trackers like Google Analytics, 
Matomo (former Piwik), mPulse, etc. to your site.
```

Comment (ashmaroli) [reply docs/_docs/plugins.md]: yes.. I'm fine with adding an adjective.. `formerly` sounds more apt though..

File: docs/_docs/plugins.md [hunk 2]
```diff
@@ -920,7 +920,7 @@ You can find a few useful plugins at the following locations:
 
 #### Other
 
-- [Analytics for Jekyll](https://github.com/hendrikschneider/jekyll-analytics) by Hendrik Schneider: An effortless way to add  various trackers like Google Analytics, Piwik, etc. to your site
+- [Analytics for Jekyll](https://github.com/hendrikschneider/jekyll-analytics) by Hendrik Schneider: An effortless way to add various trackers like Google Analytics, Matomo (former Piwik), mPulse, etc. to your site.
```
Comment (pup) [inline docs/_docs/plugins.md]: This should say `formerly` instead of `former`.

**Comment by pup** [comment]
@jekyllbot: merge +docs

**Pull request merged**